### PR TITLE
Update Index after AIP 2.0

### PR DESCRIPTION
**Pull request by swcurran**
Signed-off-by: Stephen Curran <swcurran@gmail.com>

### CODE REVIEW

File: index.md [hunk 1]
```diff
@@ -32,7 +32,7 @@
 * [0160: Connection Protocol](features/0160-connection-protocol/README.md) (2019-08-06, [5 impls](features/0160-connection-protocol/README.md#implementations) &mdash; [`feature`](/tags.md#feature) [`protocol`](/tags.md#protocol) [`test-anomaly`](/tags.md#test-anomaly))
 * [0211: Mediator Coordination Protocol](features/0211-route-coordination/README.md) (2021-03-15, [1 impl](features/0211-route-coordination/README.md#implementations) &mdash; [`feature`](/tags.md#feature) [`protocol`](/tags.md#protocol) [`test-anomaly`](/tags.md#test-anomaly))
 * [0250: Rich Schema Objects](concepts/0250-rich-schemas/README.md) (2019-10-08 &mdash; [`concept`](/tags.md#concept) [`rich-schemas`](/tags.md#rich-schemas))
-* [0302: Aries Interop Profile](concepts/0302-aries-interop-profile/README.md) (2020-01-30 &mdash; [`concept`](/tags.md#concept))
+* [0302: Aries Interop Profile](concepts/0302-aries-interop-profile/README.md) (2021-01-06 &mdash; [`concept`](/tags.md#concept))
```
Comment (dhh1128) [inline index.md:35]: Did you mean June 30, 2021 (2021-06-30), or did you really mean Jan 6, 2021?

**Review by dhh1128 — APPROVED** [review]
I'm wondering if we've got a date mistyped. Maybe, or maybe not?

**Comment by dhh1128** [comment]
I'm going to go ahead and merge this. If the date is mistyped, we can submit a follow-up PR. If not, then we've got one more thing done. :-)

**Comment by swcurran** [comment]
I think that is when the first AIP 2.0 PR went in, and it never got changed.  We need to fix it in the 302 RFC.

**Pull request merged**